src/patternfly/components/Page/page.scss

```diff
@@ -71,7 +71,7 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
 
   // Sidebar
   --pf-c-page__sidebar--ZIndex: var(--pf-global--ZIndex--sm);
-  --pf-c-page__sidebar--Width: #{pf-size-prem(290px)};
+  --pf-c-page__sidebar--Width: #{pf-size-prem($pf-global--sidebar--Width)};
```

Comment:
I think a global var for this is probably a good thing for this reason - the sidebar width is probably a useful value to have outside of the context of the page component considering things like media queries or other SCSS vars may want to reference it. And this works, I have a few thoughts though, curious what you think @mattnolting @srambach.

* I wonder if it should be a CSS var, too, though I wouldn't want to introduce a problem/breaking change with this update, since this is such an integral part of the overall layout.
  * I think the way you've done it here is probably the least intrusive, but I'm thinking you could also add a CSS var without any risk of something breaking. That would be more consistent, and could be used for other uses where a CSS var can be referenced, but isn't within `.pf-c-page` - like the `:root` scope. wdyt?
* Since the current media queries are based off of the sidebar at a `rem` width (using `pf-size-prem()`), I'm wondering if the global var should have a `rem` value so the new media queries are based off of that rem value. Probably wouldn't make a difference, but I'm guessing might be more accurate if someone as modified `$pf-global--space-size-root`? https://github.com/patternfly/patternfly/blob/b6fdfb59baeeca10a5942c66c3ae208379a20a3f/src/patternfly/sass-utilities/functions.scss#L10-L12
  * Update this to update to `$pf-global--sidebar--Width: pf-size-prem(290px) !default;`? https://github.com/patternfly/patternfly/blob/4b338526c8349c2c224d3e951ca5815aa81269e9/src/patternfly/sass-utilities/scss-variables.scss#L157